examples/supertuxkart/Dockerfile

```diff
@@ -12,36 +12,55 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# build
-FROM golang:1.13.5 as builder
+# build go
+FROM golang:1.13.5 as go-builder
 WORKDIR /go/src/supertuxkart
 
 RUN apt-get update && apt-get install -y curl tar xz-utils
 
-COPY examples/supertuxkart/main.go .
-COPY . /go/src/agones.dev/agones
+COPY main.go .
+COPY go.mod .
 RUN go build -o wrapper .
 
-RUN curl -o supertuxkart.tar.xz https://pilotfiber.dl.sourceforge.net/project/supertuxkart/SuperTuxKart/1.1/SuperTuxKart-1.1-linux.tar.xz && \
-  tar -xf supertuxkart.tar.xz
+# compile stk
+FROM debian:buster as stk-builder
+
+RUN apt update && apt-get install -y build-essential cmake libbluetooth-dev \
+libcurl4-openssl-dev libenet-dev libfreetype6-dev libharfbuzz-dev libfribidi-dev \
+libgl1-mesa-dev libglew-dev libjpeg-dev libogg-dev libopenal-dev libpng-dev \
+libssl-dev libvorbis-dev libxrandr-dev libx11-dev nettle-dev pkg-config zlib1g-dev \
+git subversion
+
+RUN git clone -b allow-ai https://github.com/markmandel/stk-code.git
+RUN svn co https://svn.code.sf.net/p/supertuxkart/code/stk-assets stk-assets
+RUN rm -rf ./stk-code/.git
+RUN rm -rf ./stk-assets/.svn
```

Comment:
Sorry I didn't catch this the first time. If you don't want the source control bits, why not `svn export`? It looks like you can [even do it for git repositories](https://stackoverflow.com/questions/160608/do-a-git-export-like-svn-export) (that are on github)!

Reply:
Ooh. Good idea. Let me take this for a spin.

At the very least, I can do a --depth on the git checkout.